staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go

```diff
@@ -37,14 +36,12 @@ type WorkEstimatorConfig struct {
 	*ListWorkEstimatorConfig     `json:"listWorkEstimatorConfig,omitempty"`
 	*MutatingWorkEstimatorConfig `json:"mutatingWorkEstimatorConfig,omitempty"`
 
-	// MinimumSeats is the minimum number of seats a request must occupy.
-	MinimumSeats uint64 `json:"minimumSeats,omitempty"`
```

Comment:
(assuming this is safe to delete as this config is not actually read by anything external to apiserver)